Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/cart/Cart.kt

```diff
@@ -185,104 +298,107 @@ fun CartItem(
     modifier: Modifier = Modifier
 ) {
     val snack = orderLine.snack
-    ConstraintLayout(
-        modifier = modifier
-            .fillMaxWidth()
-            .clickable { onSnackClick(snack.id) }
-            .padding(horizontal = 24.dp)
-    ) {
-        val (divider, image, name, tag, priceSpacer, price, remove, quantity) = createRefs()
-        createVerticalChain(name, tag, priceSpacer, price, chainStyle = ChainStyle.Packed)
-        SnackImage(
-            imageUrl = snack.imageUrl,
-            contentDescription = null,
-            modifier = Modifier
-                .size(100.dp)
-                .constrainAs(image) {
-                    top.linkTo(parent.top, margin = 16.dp)
-                    bottom.linkTo(parent.bottom, margin = 16.dp)
-                    start.linkTo(parent.start)
+    Row(modifier = Modifier.background(JetsnackTheme.colors.uiBackground)) {
```

Comment:
Doesn't look like this Row is needed? It only seems to have 1 child

Reply:
I removed the Row